python/ql/test/experimental/dataflow/validTest.py

```diff
@@ -75,6 +75,7 @@ def check_tests_valid_after_version(testFile, version):
     check_tests_valid("variable-capture.test_collections")
     check_tests_valid("variable-capture.by_value")
     check_tests_valid("variable-capture.test_library_calls")
+    check_tests_valid("variable-capture.test_fields")
```

Comment:
Does Python have its own test framework? I was expecting a normal QL (inline expectation) test.

Reply:
Yes. This is an extra layer that ensures that our test expectations agree with the Python interpreter :-)

Reply:
(The variable capture test is also slightly custom inline dataflow test, only annotating with "captured", but that is for historical reasons. We should probably convert it to a standard inline dataflow test at some point.)